### PR TITLE
fix openapi v1 router

### PR DESCRIPTION
**Pull request by sfwn**
#### What type of this PR

/kind bug

#### What this PR does / why we need it:

- openapi-ng register apiv1's router directly
- delete openapi definitions which already provided by protobuf

tested in hkci.

#### Which issue(s) this PR fixes:

- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/bug?id=222352&issueFilter__urlQuery=eyJpdGVyYXRpb25JRHMiOls1MDZdLCJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIiwiUkVTT0xWRUQiXSwiYXNzaWduZWVJRHMiOlsiOTIiXX0%3D&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTB9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6InRhYmxlIiwiY2hpbGRyZW5WYWx1ZSI6eyJrYW5iYW4iOiJkZWFkbGluZSJ9fQ%3D%3D&iterationID=506&type=BUG)

#### Specified Reviewers:

/assign @recallsong @Effet

### CODE REVIEW

File: modules/openapi/api/apis/dop/cmdb_backup_download.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ import (
 var CMDB_BACKUP_DOWNLOAD = apis.ApiSpec{
 	Path:          "/api/backup/<uuid>",
 	BackendPath:   "/api/files/<uuid>",
-	Host:          "dop.marathon.l4lb.thisdcos.directory:9527",
+	Host:          "core-services.marathon.l4lb.thisdcos.directory:9526",
```
Comment (Effet) [inline modules/openapi/api/apis/dop/cmdb_backup_download.go:26]: what backup means, and why just keep leave /api/backup/* and /api/files/* was deleted?

**Review by Effet — CHANGES_REQUESTED** [review]

File: modules/openapi/api/apis/dop/cmdb_backup_download.go (hunk 1 above)
Comment (sfwn) [reply modules/openapi/api/apis/dop/cmdb_backup_download.go:26]: Backup is a function provided by gittar.
Now it's not show on UI. So I will delete this openapi definition in this pr.

Comment (sfwn) [reply modules/openapi/api/apis/dop/cmdb_backup_download.go:26]: `/api/files` just implemented on core-services, so I delete definitions which proxy to dop, remain core-services.

**Review by Effet — APPROVED** [review]

**Pull request merged**